tests/test_conservation.py

```diff
@@ -0,0 +1,377 @@
+import climt
+from sympl import (
+    get_constant, AdamsBashforth,
+    TendencyComponent, TendencyStepper)
+from datetime import timedelta
+import numpy as np
+
+
+class Conservation(object):
```

Comment:
Rename to `ConservationTestBase`. This shouldn't be picked up by testing packages since it doesn't start or end with `Test`.

Reply:
Great use of subclassing to make these tests!